### ISSUE TITLE
Check for table_schema in commonClient.ensureTable

### ISSUE THREAD

**Issue by jontelm** [rickbergfalk/postgrator#48]
We need a check for the table_schema otherwise it's conflicts if we have more then one database that using this lib on the same server.

**and table_schema = ${config.database}**
```
const sql = `
      SELECT column_name
      FROM INFORMATION_SCHEMA.COLUMNS
      WHERE table_name = '${config.schemaTable} and table_schema = ${config.database}';
    `
```
The error I get is:
```
Error: Error: ER_NO_SUCH_TABLE: Table 'abc.schemaversion' doesn't exist
```

**Comment by rickbergfalk** [comment]
Thanks for reporting @jontelm! I've made some changes and published to npm as version 3.4.0.

**Comment by jontelm** [comment]
Thanks for a good and powerful module! @rickbergfalk
